src/vs/workbench/contrib/notebook/browser/controller/coreActions.ts

```diff
@@ -427,3 +428,9 @@ MenuRegistry.appendMenuItem(MenuId.EditorContext, {
 	group: CellOverflowToolbarGroups.Insert,
 	when: NOTEBOOK_EDITOR_FOCUSED
 });
+
+MenuRegistry.appendMenuItem(MenuId.NotebookCellTitle, {
+	title: localize('miShare', "Share"),
+	submenu: MenuId.MenubarShare,
```

Comment:
A notebook cell is in the editor. Does the `EditorContextShare` menu id make more sense here?

Reply:
Yep, `EditorContextShare` is better here.